### PR TITLE
Introduce shell functions to play with rabbit and oslo

### PR DESCRIPTION
**Pull request by 4383**
Feature:
- ban rabbitmq packets with firewall rules to emulate network issues
- restore rabbitmq packets
- setup a rabbitmq cluster with 3 nodes
- monitor cluster
- start a notifier
- start a listener

Co-authored-by: Sébastien Boyron <sebastien@boyron.eu>

### CODE REVIEW

**Comment by 4383** [comment]
@dj4ngo o/

File: env.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+function oslo_check () {
+    if [ -z "$(which docker)" ]; then
+        echo "Docker is required. Please it install first"
+        exit 1
+    fi
+}
+
+function oslo_start_cluster () {
+    oslo_check
+    if [ !-d docker-rabbitmq-cluster ]; then
```
Comment (kgiusti) [inline env.sh]: This throws a syntax error when run under bash - a space is needed between the "!" and the "-d".

File: env.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        setup_cluster
```
Comment (kgiusti) [inline env.sh]: "oslo_setup_cluster"

**Review by kgiusti — COMMENTED** [review]
Thanks for the pull request!

Hit a couple of problems when running on my fedora system - see inline.

Otherwise looks good.

**Comment by 4383** [comment]
I fixed all your comments and also added more doc.

**Comment by kgiusti** [comment]
Great, thanks.

**Pull request merged**